packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,136 @@
+# Capturing Vital Signs
+
+Vital signs are a group of important medical signs that measure the body's most vital (life-sustaining) functions. The four main vital signs are body temperature, pulse rate, respiration rate, and blood pressure. However, sometimes other measures, such as blood oxygen, weight, and blood glucose level, are included as well.
```

Comment:
WIth the proposed shift in scope, clarify that the concept of observations can include vital signs, as well as lab results, device readings, and even survey responses